practice-javascript/src/js/models/userModel.js

```diff
@@ -0,0 +1,14 @@
+class UserModel {
+  constructor(id,emai,password,first_name,last_name,avatar_url,phone_number,role){
```

Comment:
@thanhtrantruong47 Follow at least one convention, you are missing a space after `,`